setup.py

```diff
@@ -50,16 +48,7 @@ def long_description():
 
 
 def version():
```

Comment:
Method no longer needed?

Reply:
Yeah. Just missed deleting it when I decided to just have the __version__ down below.